With `-ftlo=auto` linking would emit:

```
  src/slic3r/GUI/GUI_Utils.hpp:92:25: warning:
  type ‘struct DPIAware’ violates the C++ One Definition Rule [-Wodr]

  template<class P> class DPIAware :
      public P,
      public wxInspector::wxInspectable

  src/slic3r/GUI/GUI_Utils.hpp:92:25: note:
  a type with the same name but different base type is defined
  in another translation unit

  wx/inspector/inspector.h:29:7: note:
  type name ‘wxInspector::wxInspectableImpl’ should match
  type name ‘wxInspector::wxInspectable’

  wx/inspector/inspector.h:57:7: note:
  the incompatible type is defined here
```

... but a build takes ages :/
LTO could be enabled in the CI for releases and locally for release builds.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -80,8 +80,12 @@ endif()
 
 if (DEFINED BBL_RELEASE_TO_PUBLIC)
     add_compile_definitions("BBL_RELEASE_TO_PUBLIC=${BBL_RELEASE_TO_PUBLIC}")
+    if (BBL_RELEASE_TO_PUBLIC)
+        add_compile_definitions(WXINSPECTOR_DISABLE)
+    endif ()
 else ()
     add_compile_definitions("BBL_RELEASE_TO_PUBLIC=$<CONFIG:Release>")
+    add_compile_definitions("$<$<CONFIG:Release>:WXINSPECTOR_DISABLE>")
 endif ()
 
 find_package(Git)

--- a/src/libslic3r/Technologies.hpp
+++ b/src/libslic3r/Technologies.hpp
@@ -51,9 +51,4 @@
 // Enable extension of tool position imgui dialog to show actual speed profile
 #define ENABLE_ACTUAL_SPEED_DEBUG 1
 
-// Disable layout inspector for public release
-#if BBL_RELEASE_TO_PUBLIC
-#define WXINSPECTOR_DISABLE
-#endif
-
 #endif // _prusaslicer_technologies_h_

